components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -188,6 +206,28 @@ class SitePermissionsFeature(
         )
     }
 
+    @SuppressLint("VisibleForTests")
+    private fun createPromptForMicrophonePermission(
+        context: Context,
+        uriString: String,
+        buttons: List<DoorhangerPrompt.Button>
+    ): DoorhangerPrompt {
+
+        val title = context.getString(R.string.mozac_feature_sitepermissions_microfone_title, uriString)
+        val optionTitle = context.getString(R.string.mozac_feature_sitepermissions_option_microphone_one)
+        val drawable = ContextCompat.getDrawable(context, R.drawable.mozac_ic_microphone)
+
+        val microphoneRadioButton = RadioButton(optionTitle).apply { checked = true }
```

Comment:
nit: No need to apply "checked" here. The first radio button in a group is always checked.

Reply:
Awesome!